server/models/migrations/next/migrate.sql

```diff
@@ -275,4 +275,89 @@ INSERT INTO `unit` VALUES
 INSERT INTO `unit` VALUES 
 (238, 'Indicators report', 'TREE.INDICATORS_REPORT', 'Rapport sur les indicateurs', 144,'/modules/reports/indicatorsReport', '/reports/indicatorsReport');
 INSERT INTO `report` (`id`, `report_key`, `title_key`) 
-VALUES  (31, 'indicatorsReport', 'TREE.INDICATORS_REPORT');
\ No newline at end of file
+VALUES  (31, 'indicatorsReport', 'TREE.INDICATORS_REPORT');
+
+
+
+
+-- jeremie lodi
+DELIMITER $$
+
+-- this is handled on change rate changed
+DROP PROCEDURE IF EXISTS `UpdateLossAccount`$$
+CREATE   PROCEDURE `UpdateLossAccount`(
```

Comment:
I agree with @mbayopanda's analysis that this name should talk about both gain or loss.  Something like `UpdateGainOrLossOnExchangeAccounts` would be preferable.